streams/src/test/java/org/apache/kafka/test/KStreamTestDriver.java

```diff
@@ -225,20 +225,9 @@ public ProcessorNode processor(String name) {
     }
 
     public void flushState() {
-        final ProcessorNode current = currNode;
-        try {
-            for (StateStore stateStore : context.allStateStores().values()) {
-                final ProcessorNode processorNode = topology.storeToProcessorNodeMap().get(stateStore);
-                if (processorNode != null) {
-                    currNode = processorNode;
-                }
-                stateStore.flush();
-            }
-        } finally {
-            currNode = current;
-
+        for (StateStore stateStore : context.allStateStores().values()) {
```

Comment:
Why do we not need `currNode` to map to the store we are flushing any more? I don't see any change in `StateStore#flush()` -- or did we not need this in the first place and this is just a cleanup of unnecessary code?